scripts/delete-user.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+
+const readline = require("readline");
+
+const DB = require("../db/DB");
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+(async () => {
+  rl.question("What FXA primary email address? ", first_answer => {
+    rl.question("Please re-type the email address to confirm. ", async (second_answer) => {
+      if (first_answer !== second_answer) {
+        console.error("Email addresses do not match.");
+        process.exit();
+      }
+      const subscriber = await DB.getSubscriberByEmail(second_answer);
+      await DB.deleteEmailAddressesByUid(subscriber.id);
```

Comment:
Do we need to check if the subscriber was found and `subscriber` isnt null/undefined before trying to yank it's `id` and `fxa_uid` properties? Maybe not since this is just a CLI tool and probably doesnt hurt anything if we get a runtime error of accessing props on undefined objects.